### PR TITLE
SQL Catch

### PR DESCRIPTION
**Pull request by MTS11648**
Don't know much about SQL at all really, but this SHOULD work.

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Later, I want to add in an extra text file for logging errors like this.

**Comment by SpadesNeil** [comment]
I'm going to run a version of this later that intentionally screws up the SQL address, just to make sure it works as expected, and then I'll undo that screw up and see if the data sends correctly.

I want to make absolutely sure we don't break everything by messing with SQL.

File: AutoBotCSharp/Agent.cs [hunk 1]
```diff
@@ -3243,12 +3244,25 @@ public static bool UpdateDBase(string command)
...
-
+                    var fileLines = System.IO.File.ReadAllLines(path);
+                    if (File.Exists(path))
+                    {
+                        foreach (var line in fileLines)
+                        {
+                            myConnection.Open();
+                            Add.ExecuteNonQuery();
+                            myConnection.Close();
+                            Console.WriteLine("SENT PREVIOUSLY ORPHANED MSQL STRING " + command);
```
Comment (SpadesNeil) [inline AutoBotCSharp/Agent.cs]: There needs to be a line here that removes this file after it is done running.

**Comment by MTS11648** [comment]
@SpadesNeil oh yeah, forgot it would be absolutely critical to delete old orphan leads. good thing you caught that.

**Comment by SpadesNeil** [comment]
Also, is there any reason why it using AppendAllText() rather than WriteLine()? Although I do not think WriteLine() will create a new file if one does not exist.

**Comment by MTS11648** [comment]
@SpadesNeil I don't know, I've just always used AppendAllText().

**Comment by SpadesNeil** [comment]
Just looked it up to make sure it wasn't what I thought it was. It's all good. I thought it was going to _overwrite_ the file. That isn't the case.

**Comment by SpadesNeil** [comment]
Removing blocker tag because there's ways around this now.

**Comment by MTS11648** [comment]
closed because i did something stupid

**Pull request closed without merging**